.github/workflows/generate-token-lists.yml

```diff
@@ -56,27 +56,66 @@ jobs:
       matrix:
         commands:
           - name: Arb1 FullList
-            command: yarn fullList --l2NetworkID 42161 --newArbifiedList ./src/ArbTokenLists/arbed_full.json --skipValidation
+            paths:
+              - ArbTokenLists/arbed_full.json
+            version: false
+            command: yarn fullList --l2NetworkID 42161 --newArbifiedList ./src/ArbTokenLists/arbed_full.json --skipValidation --ignorePreviousList
```

Comment:
since it isn't versioned anyway, I think fullList could just ignore-previous by default (this is fine tho)